opus/import/main_opus_import.py

```diff
@@ -319,6 +324,8 @@
     impglobals.ARGUMENTS.do_import = True
     impglobals.ARGUMENTS.copy_import_to_permanent_tables = True
     impglobals.ARGUMENTS.drop_new_import_tables = True
+    # Comment/Uncomment the line below to determine if row files are used
+    impglobals.ARGUMENTS.import_use_row_files = True
```

Comment:
This shouldn't be here...it eliminates the benefit of having the option in the first place